src/features/basic/prodco-order-eta.ts

```
@@ -2,15 +2,15 @@ import { productionStore } from '@src/infrastructure/prun-api/data/production';
 import { formatEta } from '@src/utils/format';
 import { refAttributeValue } from '@src/utils/reactive-dom';
 import { createReactiveSpan } from '@src/utils/reactive-element';
-import { observeChildListChanged } from '@src/utils/mutation-observer';
+import { observeDescendantAttributeListChanged } from '@src/utils/mutation-observer';
 
 function onTileReady(tile: PrunTile) {
   const line = computed(() => productionStore.getById(tile.parameter)!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)